packages/next-swc/crates/core/src/styled_jsx/transform_css.rs

```diff
@@ -19,6 +21,8 @@ pub fn transform_css(
     is_global: bool,
     class_name: &Option<String>,
 ) -> Result<Expr, Error> {
+    debug!("CSS: \n{}", style_info.css);
```

Comment:
This expression (and corresponding expression statement) will be completely noop on release builds.

Reply:
cc @padmaia 
It was useful for debugging to me, but I'm not sure if it's good to have this line.

Reply:
We enable `release_max_level_off ` of [`tracing`](https://docs.rs/crate/tracing/0.1.29/features), which is a crate for logging.
The cargo feature `release_max_level_off` removes all logging for release build and all macro calls for logging become completely no-op.

Reply:
This logging statement will print the input passed to the css parser while testing.
`testing` is a crate for testing swc crates and `next-swc`, and it configures logger by default.

Logging level is `debug` by default, but if you run it like `RUST_LOG=trace cargo test`, `trace!` calls will also print the messages.

Reply:
As a side note, `tracing` is used by swc and rustc.

Reply:
![image](https://user-images.githubusercontent.com/29931815/145156948-8612834c-d530-4ac0-84e9-7b6714f87b85.png)